components/Forms/validators.ts

```diff
@@ -18,6 +18,9 @@ export const validateSubmission = (values: Submission) => {
     if (!!url && url?.protocol !== "http:" && url?.protocol !== "https:") {
       errors.mediaURI = "Only http or https";
     }
+    if (await urlDuplicate(url)){
+      errors.mediaURI = "Duplicate submission";
+    }
```

Comment:
We should only run this check if `errors.mediaURI` isn't set already, that way we avoid making the call to supabase.